.travis.yml

```
@@ -24,7 +24,7 @@ install:
   - python setup.py install
 
 script:
-    - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then py.test partd --doctest-modules --verbose; else py.test partd --verbose; fi
+    - py.test partd --verbose
```

Nitpick: could we keep the doctests and run them on the 3.7 build instead of the (not non-existent) 2.7 build

```suggestion
    - if [[ $TRAVIS_PYTHON_VERSION == '3.7' ]]; then py.test partd --doctest-modules --verbose; else py.test partd --verbose; fi
```